File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-restore-selections-after-searches_2.vb
        searchArea.Find.ClearFormatting()
        searchArea.Find.Text = "find me"

        searchArea.Find.Replacement.ClearFormatting()
        searchArea.Find.Replacement.Text = "Found"

        searchArea.Find.Execute(Replace:=Word.WdReplace.wdReplaceAll)

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-restore-selections-after-searches_2.cs
            searchArea.Find.ClearFormatting(); 
            searchArea.Find.Text = "find me"; 

            searchArea.Find.Replacement.ClearFormatting(); 
            searchArea.Find.Replacement.Text = "Found"; 

            object replaceAll = Word.WdReplace.wdReplaceAll; 

            searchArea.Find.Execute(
                ref missing, ref missing, ref missing, ref missing, ref missing,
                ref missing, ref missing, ref missing, ref missing, ref missing,
                ref replaceAll, ref missing, ref missing, ref missing, ref missing);